numpyro/distributions/continuous.py

```diff
@@ -1075,6 +1075,132 @@ def cdf(self, x):
         return self.base_dist.cdf(jnp.log(x))
 
 
+def _batch_solve_triangular(A, B):
+    """Extende solve_triangular for the case that B.ndim > A.ndim.
+        This is achived by first flattening the leadind B.ndim - A.ndim axes of B and
+        then moving them to the end.
+
+    Args:
+        A (_type_): A triangular array.
+        B (_type_): _description_
```

Comment:
Could you update docstring here and the MatrixNormal distribution? You can use `jnp.ndarray` for `_type_`. It would be nice to use
```
:param: ...
:return: ...
```
for consistency (as in [here](https://github.com/pyro-ppl/numpyro/blob/7f7e71077014813e06eb214728b7d6feebcb7f67/numpyro/distributions/mixtures.py#L27-L35))

Reply:
Like this? 
```
:param jnp.ndarray (...,M,M) A: An array with lower triangular structure in the last two dimensions.
:param jnp.ndarray (...,M,) or (...,M,N) B: Right-hand side matrix in A x = B.
```

Reply:
Yup